model_analyzer/config/generate/optuna_run_config_generator.py

```diff
@@ -435,8 +436,16 @@ def _create_model_run_config(
         model_config_variant: ModelConfigVariant,
         trial_objectives: TrialObjectives,
     ) -> ModelRunConfig:
+        trial_batch_size = (
+            int(trial_objectives["batch_sizes"])
+            if "batch_sizes" in trial_objectives
+            else DEFAULT_BATCH_SIZES
+        )
         perf_analyzer_config = self._create_perf_analyzer_config(
-            model.model_name(), model, int(trial_objectives["concurrency"])
+            model.model_name(),
```

Comment:
I'm not a fan of 4 args to a function where you have to know the exact order to pass them in. At a minimum it should be named args.

Reply:
I've made them named args.